tests/unit_tests/pyanaconda_tests/modules/payloads/source/test_source_live_os.py

```diff
@@ -189,14 +214,18 @@ def test_setup_install_source_task_name(self):
 
     @patch("pyanaconda.modules.payloads.source.live_os.initialization.blivet.util.mount")
     @patch("pyanaconda.modules.payloads.source.live_os.initialization.stat")
+    @patch("pyanaconda.modules.payloads.source.live_os.initialization.SetUpLiveOSSourceTask"
+           "._calculate_required_space")
     @patch("os.stat")
     @patch_dbus_get_proxy
-    def test_setup_install_source_task_run(self, proxy_getter, os_stat, stat, mount):
+    def test_setup_install_source_task_run(self, proxy_getter, os_stat, required_space,
```

Comment:
In this test, you should check the return value of `SetUpLiveOSSourceTask(...).run()`. Make sure it is an instance of `SetupLiveOSResult` and the required space is set to the expected value.